subxt/src/blocks/blocks_client.rs

```diff
@@ -127,7 +127,7 @@ where
         let client = self.client.clone();
         header_sub_fut_to_block_sub(self.clone(), async move {
             let sub = client.backend().stream_finalized_block_headers().await?;
-            BlockStreamRes::Ok(sub)
+            BlockStreamRes::Ok(sub.stream.expect("TODO use retry subscription here???"))
```

Comment:
Why would we need to do anything special here? Doesn't the backend handle any retry stuff internally anyways (assuming a retry RPC client is used)?

Reply:
yes correct